TAP version 13
ok 1 - expect truthy value
# Subtest: plans of 1
    ok 1 - before sync thrower
    # Subtest: sync thrower
        1..1
        ok 1 - before the bomb
    ok 2 - sync thrower ___/# time=[0-9.]+(ms)?/~~~
    
    not ok 3 - Error: pwnSync
      ---
      {"at":{"column":11,"file":"test/test/throws-and-plans.js","line":10},"message":"Error: pwnSync","source":"throw new Error('pwnSync')\n","test":"sync thrower"}
      ...
    
    # Subtest: async thrower
        1..3
        ok 1 - before set the bomb
        ok 2 - after set the bomb
        ok 3 - before the bomb
    ok 4 - async thrower ___/# time=[0-9.]+(ms)?/~~~
    
    ok 5 - after child
    1..5
    # failed 1 of 5 tests
not ok 2 - plans of 1 ___/# time=[0-9.]+(ms)?/~~~

# Subtest: no assert only throw
    not ok 1 - AssertionError: false is truthy right?
      ---
      {"actual":false,"at":{"column":3,"file":"test/test/throws-and-plans.js","line":29},"expected":true,"generatedMessage":false,"message":"AssertionError: false is truthy right?","name":"AssertionError","operator":"==","source":"assert(false, 'false is truthy right?')\n","test":"no assert only throw"}
      ...
    
    1..1
    # failed 1 of 1 tests
not ok 3 - no assert only throw ___/# time=[0-9.]+(ms)?/~~~

# Subtest: plans of 8
    ok 1 - before child
    # Subtest: sync thrower
        1..8
        ok 1 - before the bomb
        not ok 2 - Error: pwnSync
          ---
          {"at":{"column":11,"file":"test/test/throws-and-plans.js","line":39},"message":"Error: pwnSync","source":"throw new Error('pwnSync')\n","test":"sync thrower"}
          ...
        
        not ok 3 - missing test
          ---
          {"at":{"column":5,"file":"test/test/throws-and-plans.js","line":35},"count":2,"missing":6,"plan":8,"source":"t.test('sync thrower', function (tt) {\n"}
          ...
        
        not ok 4 - missing test
        not ok 5 - missing test
        not ok 6 - missing test
        not ok 7 - missing test
        not ok 8 - missing test
        # failed 7 of 8 tests
    not ok 2 - sync thrower ___/# time=[0-9.]+(ms)?/~~~
    
    # Subtest: async thrower
        1..8
        ok 1 - before set the bomb
        ok 2 - after set the bomb
        not ok 3 - Error: pwn
          ---
          {"at":{"column":13,"file":"test/test/throws-and-plans.js","line":18},"message":"Error: pwn","source":"throw new Error('pwn')\n","test":"async thrower"}
          ...
        
        not ok 4 - missing test
          ---
          {"at":{"column":5,"file":"test/test/throws-and-plans.js","line":46},"count":3,"missing":5,"plan":8,"source":"t.test('async thrower', function (tt) {\n"}
          ...
        
        not ok 5 - missing test
        not ok 6 - missing test
        not ok 7 - missing test
        not ok 8 - missing test
        # failed 6 of 8 tests
    not ok 3 - async thrower ___/# time=[0-9.]+(ms)?/~~~
    
    ok 4 - after child
    1..4
    # failed 2 of 4 tests
not ok 4 - plans of 8 ___/# time=[0-9.]+(ms)?/~~~

not ok 5 - Error: pwn
  ---
  {"at":{"column":13,"file":"test/test/throws-and-plans.js","line":51},"message":"Error: pwn","source":"throw new Error('pwn')\n","test":"TAP"}
  ...

1..5
# failed 4 of 5 tests
___/# time=[0-9.]+(ms)?/~~~

